yaml/pois.yaml

```diff
@@ -1812,3 +2106,32 @@ filters:
       <<: *output_properties
       kind: attraction
       tier: 6
+
+  # if a more specific shop isn't found, then default to generic shop kind
+  - filter:
+      shop: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: shop
+  # if a more specific craft isn't found, then default to generic craft kind
+  - filter:
+      craft: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: craft
+  # if a more specific off isn't found, then default to a generic office.
+  - filter:
+      office: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: office
+  # if a more specific industrial isn't found, then default to generic industrial kind
+  - filter:
+      industrial: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: industrial
```

Comment:
Please ensure test and documentation for `industrial` fallback.

Reply:
Added in dcce2ee.

Side note: Top values for `industrial` that we don't currently map to a kind:

* `oil`
* `wellsite`
* `factory`
* `well_cluster`
* `warehouse`
* `brickyard`
* `gas`
* `timber`
* `scrap_yard`
* `port` (although we do have `port_terminal` and `harbour`)
* `brickworks`
* `depot`
* `auto_wrecker`
* `fracking`
* `sawmill`
* `refinery`
* `salt_pond`
* `distributor`
* `brewery` (although we do have a kind for `craft=brewery`)